packages/components/src/DateTimePickers/DateTime/Manager/Manager.component.js

```diff
@@ -50,7 +50,9 @@ class ContextualManager extends React.Component {
 	constructor(props) {
 		super(props);
 
-		checkSupportedDateFormat(props.dateFormat);
+		if (props.dateFormat) {
```

Comment:
You can revert that, we always have a dateFormat, if not provided, we have a default one